content/en/news/releases/1.10.x/announcing-1.10/change-notes/index.md

```diff
@@ -0,0 +1,179 @@
+`--set values.pilot.env.PILOT_JWT_PUB_KEY_REFRESH_INTERVAL=<duration>` during installation.
+The default interval is `20m`. Valid time units are "ns", "us", "ms", "s", "m", "h".
+
+- **Added**  istiod JWT public key refresh job will now retry the failed fetch of the `jwks_uri` with exponential backoff.
+  ([Issue #30261](https://github.com/istio/istio/issues/30261))
+
+- **Fixed** removed ability to configure `trustDomain` from Helm `global.values`. Now it is configured through `meshConfig.trustDomain` ([Issue #27734](https://github.com/istio/istio/issues/27734))
+
+- **Fixed** an issue causing simple TLS traffic to ports not exposed by a `Service` to be rejected by servers when in `PERMISSIVE` mTLS mode.
+  ([Issue #31297](https://github.com/istio/istio/issues/31297))
+
+## Telemetry
+
+- **Added** experimental support for the Telemetry API.
+  ([Issue #24284](https://github.com/istio/istio/issues/24284))
+
+- **Fixed** missing `destination_cluster` label at metric reported by client proxy on request failures.
+  ([Issue #29373](https://github.com/istio/istio/issues/29373))
+
+- **Fixed** an issue where Envoy did not start up properly when duplicate stats tags were configured.
+  ([Issue #31270](https://github.com/istio/istio/issues/31270))
+
+## Extensibility
+
+- **Added** reliable Wasm module remote load with istio-agent.
+  ([Issue #29989](https://github.com/istio/istio/issues/29989))
+
+## Installation
+
+- **Added** `istioctl experimental revision tag` command group. Revision tags act as aliases for
+control plane revisions. Users can label their namespaces with a revision tag rather than pointing them
+directly at a revision and selectively decide the granularity of their namespace labels. This makes it possible
+to perform upgrades with the ease of in-place upgrades while having the safety revision-based upgrades
+under the hood. Read more about using revision tags [here](/docs/setup/upgrade/canary/#stable-revision-labels-experimental).
+
+- **Improved** `ConfigMaps` to be read directly rather than from volume mounts. This improves the speed
+of updates and ensures that for external istiod installations that the configmaps are read from the config cluster.
+  ([Issue #31410](https://github.com/istio/istio/issues/31410))
+
+- **Improved** the sidecar injector to better utilize pod labels to determine if injection is required.
+  ([Issue #30013](https://github.com/istio/istio/issues/30013))
+
+- **Updated** non-revisioned installs to target the label `istio.io/rev=default` for injection in addition to the
+existing default injection labels (`istio-injection=enabled` and `sidecar.istio.io/inject=true`).
+
+- **Added** support for slash characters in environment variables on `injectionURL`.
+  ([Issue #31732](https://github.com/istio/istio/issues/31732))
+
+- **Added** an `external` profile for installing Istio with an external control plane and deprecated the `remote` profile.
```

Comment:
Do we need to add this deprecation above?

Reply:
Good call, done